README.md

```diff
@@ -42,11 +42,13 @@ Use these instructions to build a clio executable from source. These instruction
   # -------------------------------
   export BOOST_ROOT=/home/my_user/boost_1_75_0
   source ~/.profile
```

Comment:
What is this trying to accomplish?
It's unnecessary.

Reply:
I don't see a reason to leave the line with `apt update/upgrade` in - I agree users can/should be able to figure that out on their own.

What about just using `$HOME` consistently? I suppose `~` would be more concise, though?

```                                                                                                                                                                                                                
# Update software & install dependencies                                                                                                                                                                           
  sudo apt-get -y install git pkg-config protobuf-compiler libprotobuf-dev libssl-dev wget build-essential bison flex autoconf cmake                                                                               
                                                                                                                                                                                                                   
# Compile Boost                                                                                                                                                                                                    
  wget -O $HOME/boost_1_75_0.tar.gz https://boostorg.jfrog.io/artifactory/main/release/1.75.0/source/boost_1_75_0.tar.gz                                                                                           
  tar xvzf $HOME/boost_1_75_0.tar.gz                                                                                                                                                                               
  cd $HOME/boost_1_75_0                                                                                                                                                                                            
  ./bootstrap.sh                                                                                                                                                                                                   
  ./b2 -j$((`nproc`+1))                                                                                                                                                                                            
  echo "export BOOST_ROOT=$HOME/boost_1_75_0" >> $HOME/.profile && source $HOME/.profile                                                                                                                           
                                                                                                                                                                                                                   
# Clone the Clio Git repository & build Clio                                                                                                                                                                       
  cd $HOME                                                                                                                                                                                                         
  git clone https://github.com/XRPLF/clio.git                                                                                                                                                                      
  cd $HOME/clio                                                                                                                                                                                                    
  cmake -B build && cmake --build build --parallel $(nproc)                                                                                                                                                        
```

Reply:
Beautiful.
``./b2 -j$((`nproc`+1))`` is still there.

Reply:
Remind me again what to replace that call to b2 with? I can't seem to find your original comment about it.

Reply:
Oh I just meant the mixed backticks and $() in one place and just $() in another.

Reply:
I suppose it's probably safer to build boost w/ `nproc`, not `nproc + 1`? So change it to `./b2 -j$(nproc)`?

Reply:
Ok, I think I've pushed all those changes now, if you want to review. Thanks again!

Reply:
I love it.